content/docs/guides/javascript/js-api.md

```diff
@@ -116,6 +116,19 @@ The following methods and configuration options accept a function taking a `Cont
 - `preOps()`
 - `postOps()`
 
+## Referencing includes files
+
+Just like in `.sqlx` files, you can reference any `includes` function, macro or constant inside a `.js` file. However, the syntax differs slightly.
+
+For example, if you would like to reference `SERIVCE_NAME` and `SERVICE_ID` from the `includes/service` file, you must declare these constants using the JavaScript builtin `require` function.
+
+```js
+const {SERIVCE_NAME, SERVICE_ID} = require("../../includes/service");
```

Comment:
it's something we want to fix, but right now it's indeed frustrating that you need to put these `../` parts into `require` calls. i'd suggest though, only including a single one (thus assuming that this file is a top-level `definitions` file), or clarifying that this is a relative path between importer and importee.

(also this example may change as a result of the clarification I've included above, as this file - since it's top-level - *should* already be automatically imported)